chunk_downloader.go

```diff
@@ -106,7 +106,7 @@ func (scd *snowflakeChunkDownloader) start() error {
 		// if the rowsetbase64 retrieved from the server is empty, move on to downloading chunks
```

Comment:
@sfc-gh-pfus We were not able to reproduce it, since it is difficult to wait until it gets to chunk downloader and then cancel the connection. We had to use the stacktrace and force the driver to cancel inside the decodeChunk() function to replicate the issue.